public/js/get_events.js

```diff
@@ -8,6 +8,8 @@ var getEventsModule = function (queryParams, callback) {
     url = baseUrl + '/?date_from=' + queryParams.date_from + '&date_to=' + queryParams.date_to;
   } else if (queryParams.date_from && !queryParams.date_to) {
     url = baseUrl + '/?date_from=' + queryParams.date_from;
+  } else {
+    url = baseUrl + '/?date_to=' + queryParams.date_to;
   }
```

Comment:
Realised there's no need to have two separate variables - url and baseUrl, just use one of them, and then you can get rid of the first if statement too